dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/EnhancedResultsFigureGenerator.java

```diff
@@ -104,16 +125,22 @@ private static String generate18(String projectName, double[] percentages, doubl
     /*
      * a private method that generates Figure 19
      */
-    private static String generate19(String projectName, double[] orig_values, double[] time_values, double orig_value,
-            List<GeometricMeanData> fig19GeoData) {
+    private static String generate19(String projectName, double[] orig_values, double[] time_values,
+            double orig_time_value, List<GeometricMeanData> fig19GeoData, boolean[] nonZero, double orig_apfd_value) {
         String result = projectName;
         double calc1 = 0;
         double calc2 = 0;
         double calc3 = 0;
```

Comment:
Please use descriptive variable names; these don't give a sense of what they stand for, making it hard for a reader to understand the code and evaluate whether it is correct.